clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonClusterProvider.groovy

```diff
@@ -148,6 +145,9 @@ class AmazonClusterProvider implements ClusterProvider<AmazonCluster> {
       loadBalancers = translateLoadBalancers(allLoadBalancers)
       targetGroups = translateTargetGroups(allTargetGroups)
       serverGroups = translateServerGroups(allServerGroups)
+    } else {
+      Collection<CacheData> allServerGroups = resolveRelationshipDataForCollection(clusterData, SERVER_GROUPS.ns, RelationshipCacheFilter.include(LAUNCH_CONFIGS.ns))
```

Comment:
I think its useful to include launch config anyway because that brings in buildInfo (or maybe we need to also include IMAGE relationship to get that)...